modules/installation-localzone-generate-k8s-manifest.adoc

```diff
@@ -26,6 +26,29 @@ $ ./openshift-install create manifests --dir <installation_directory> <1>
 <1> For `<installation_directory>`, specify the installation directory that
 contains the `install-config.yaml` file you created.
 
+. Set the MTU size to 1200
+
+> TODO: Steps described on the official doc https://docs.openshift.com/container-platform/4.12/networking/changing-cluster-network-mtu.html#mtu-value-selection_changing-cluster-network-mtu
+
+> TODO: Do we need to add a pop-up explaining that it's a requirement of AWS to communicate withing the nodes in Local Zones and regular zones?
+
+> TODO: check the impact on cluster. Make sure we are validating this on the tests.
```

Comment:
Those items should be resolved with the teams.

> 1300 bytes between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region.

[AWS recommends](https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html) 1300 to communicate with regular zones, we are decreasing the overhead for OVN (100) as per the recommendation on our docs.